config/dataplex/config.go

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+//
+// SPDX-License-Identifier: CC0-1.0
+
+package dataplex
+
+import (
+	"github.com/crossplane/upjet/pkg/config"
+)
+
+// Configure configures individual resources by adding custom
+// ResourceConfigurators.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("google_dataplex_aspect_type", func(r *config.Resource) {
+		r.References["project"] = config.Reference{
+			TerraformName: "google_project",
+		}
+		r.MarkAsRequired("aspect_type_id")
```

Comment:
Since we moved the `aspect_type_id` field to `metadata.name` due to external-name configuration, no need this configuration. Could you please remove?

Reply:
I removed the configuration to be able to add the release that will be made today.